lib/fluent/plugin/in_systemd.rb

```diff
@@ -76,12 +75,9 @@ def formatted(entry)
       end
 
       def watch
-        while @running
-          next unless @journal.wait(1_000_000)
```

Comment:
I think this is fine ... we can always back out on this if anyone has specific issues.